src/components/Map/MapComponent.tsx

```diff
@@ -0,0 +1,22 @@
+import 'leaflet/dist/leaflet.css';
+import React from 'react';
```

Comment:
```suggestion
```
Remove these imports. The `css` file is not used (there are no changes when I remove this import). 
`React` doesn't have to be imported. It had to in previous versions of the React library but it is no longer necessary.